change in SAS Token management

axelgMS opened the pull request:
CRI 51437740 

There's a change in the way Azure Storage manages SAS token for Table service
Documented here https://github.com/Azure/azure-cli/issues/4827


If we use "az storage account generate-sas --services bt" - then the generated SAS token will only include Blob service (ss=b).
We need to generate the SAS token for the Table service here, so we need to use "az storage table generate-sas --services bt" and that will generate a SAS token for Table service too (ss=bt).

rmca14 commented:
Please update the "Note" content with the proper syntax. For guidance, see https://github.com/Azure/azure-content-pr/blob/master/contributor-guide/custom-markdown-extensions.md

Please update with the proper link syntax. For guidance, see https://github.com/Azure/azure-content-pr/blob/master/contributor-guide/create-links-markdown.md

Thank you

torsteng commented:
Since the link in the "Note" content does not refer to a documentation page, I applied the guidance copied below. Let me know if this is not correct. 

I committed the corresponding change to this pull request.

Thanks,
Torsten

To link to a page on ACOM (such as a pricing page, SLA page or anything else that is not a documentation article), use an absolute URL, but omit the locale. The goal here is that links work in GitHub and on the rendered site:
[link text](http://azure.microsoft.com/pricing/details/virtual-machines/)

rmca14 commented:
Also insert the proper "Note" syntax, which is to insert the following in front of the Note text:

> [AZURE.NOTE]

Guidance here, https://github.com/Azure/azure-content-pr/blob/master/contributor-guide/custom-markdown-extensions.md

Thank you

torsteng commented:
Done. I also adjusted another 'note' paragraph to be consistent across the doc.

rmca14 commented:
Thank you

axelgMS committed:
change in SAS Token management

CRI 51437740 

There's a change in the way Azure Storage manages SAS token for Table service
Documented here https://github.com/Azure/azure-cli/issues/4827


If we use "az storage account generate-sas --services bt" - then the generated SAS token will only include Blob service (ss=b).
We need to generate the SAS token for the Table service here, so we need to use "az storage table generate-sas --services bt" and that will generate a SAS token for Table service too (ss=bt).
